### PR TITLE
Fix basic errors in integration/run_tests.py

### PR DESCRIPTION
**Pull request by arostamianfar**
Note: The tests still do not run successfully e2e (table deletion fails when cleaning up), but they progress further :)

Issue #62

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
     # This is needed due to https://issues.apache.org/jira/browse/BEAM-3357.
     'grpcio>=1.3.5,<=1.7.3',
     'apache-beam[gcp]>=2.2',
+    'google-api-python-client>=1.6',
```
Comment (bashir2) [inline setup.py]: nit: Do you mind adding a note with a reference to Issue #71 here?

File: setup.py [hunk 2]
```diff
@@ -21,6 +21,8 @@
...
+    'google-cloud-bigquery>=0.25',
```
Comment (bashir2) [inline setup.py]: Are you sure that version 0.25 works or should this be > 0.25 (see my TODO above)?

File: gcp_variant_transforms/testing/integration/run_tests.py [hunk 3]
```diff
@@ -129,12 +146,13 @@ def __init__(self, args):
...
     self.credentials = GoogleCredentials.get_application_default()
-    client = bigquery.Client(project=self.project)
     self.dataset_id = 'integration_tests_{}'.format(
         datetime.now().strftime('%Y%m%d_%H%M%S'))
+    self.dataset = None
+
+  def __enter__(self):
+    client = bigquery.Client(project=self.project)
     dataset_ref = client.dataset(self.dataset_id)
     # TODO(bashir2): There is a mismatch between the bigquery API versions
```
Comment (bashir2) [inline gcp_variant_transforms/testing/integration/run_tests.py]: I think you can remove this TODO now. BTW, I thought we need a version strictly greater than 0.25.

Comment (arostamianfar) [reply gcp_variant_transforms/testing/integration/run_tests.py]: Done. And good catch! I changed to be >0.25.

File: setup.py (hunk 2 above)
Comment (arostamianfar) [reply setup.py]: Nice catch. Done. It installed v0.29.0 on my machine.

File: setup.py (hunk 1 above)
Comment (arostamianfar) [reply setup.py]: Done.

File: setup.py [hunk 4]
```diff
@@ -21,6 +21,11 @@
     # This is needed due to https://issues.apache.org/jira/browse/BEAM-3357.
     'grpcio>=1.3.5,<=1.7.3',
     'apache-beam[gcp]>=2.2',
+    # Note that adding 'google-api-python-client>=1.6' causes some dependency
+    # mismatch issues if using 'setup.py install'. See Issue #71.
```
Comment (bashir2) [inline setup.py]: nit: dependency issues are there no matter we use `setup.py install` or not, the difference seems to be that `setup.py install` stops while `pip install` continues but still installs conflicting versions.

Comment (arostamianfar) [reply setup.py]: Done.

**Pull request merged**